src/Transports.AspNetCore/IUserContextBuilder.cs

```diff
@@ -15,7 +13,7 @@ public interface IUserContextBuilder
     /// <param name="payload">
     /// The payload of the WebSocket connection request, if applicable.  Typically this is either <see langword="null"/> or
     /// an object that has not fully been deserialized yet; when using the Newtonsoft.Json deserializer, this will be a JObject,
-    /// and when using System.Text.Json this will be a <see cref="System.Text.Json.JsonElement">JsonElement</see>.  You may call
+    /// and when using System.Text.Json this will be a JsonElement.  You may call
```

Comment:
cred removed?

Reply:
No reference available in .Net standard 2.0